app/src/main/java/com/project/pv239/customtimealarm/fragments/MainFragment.java

```diff
@@ -44,16 +48,33 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         mUnbinder = ButterKnife.bind(this, view);
         mAdapter = new AlarmsAdapter(new ArrayList<Alarm>(), this);
         mList.setAdapter(mAdapter);
-        mList.setLayoutManager(new LinearLayoutManager(getContext()));
-
+        LinearLayoutManager manager = new LinearLayoutManager(getContext());
+        mList.setLayoutManager(manager);
+        DividerItemDecoration dividerItemDecoration = new DividerItemDecoration(mList.getContext(),
+                manager.getOrientation());
+        mList.addItemDecoration(dividerItemDecoration);
```

Comment:
mozno by pre citelnost kodu nebolo odveci to rozdelit do par mensich metod z priznacnym nazovm ktore sa z onCreate iba volaju :) najblizsie ked budeme v tejto metode robit nieco tak to refactorneme :D

Reply:
👍